src/app/generate/generate.go

```diff
@@ -182,7 +197,7 @@ func Generate(cCtx *cli.Context) error {
 	return nil
 }
 
-func addDepSource(cCtx *cli.Context, sources []changelog.Source, appendDep appendDepSrc) ([]changelog.Source, error) {
+func addDepSource(cCtx *cli.Context, sources []changelog.Source, appendDep appendDepSrc, deps []string) ([]changelog.Source, error) {
```

Comment:
```suggestion
func addDepSource(cCtx *cli.Context, sources []changelog.Source, appendDep appendDepSrc, excludedDeps []string) ([]changelog.Source, error) {
```

Reply:
Done in 4ffdd63741f6aca325bc28144f13abf165546015.